Makefile

```diff
@@ -27,3 +27,11 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 quality: clean ## check coding style with pycodestyle and pylint
 	pycodestyle ./openedx_external_enrollments
```

Comment:
Could you add the pycodestyle configuration? https://github.com/eduNEXT/cookiecutter-openedx-plugin/pull/5/files#diff-8f7dc0f5e25c8f49742a1c4d1feaddf2R282
I know it's not part of this PR. @andrey-canon

Reply:
Why do you think that is necessary ? The max length is already set, but I don't know the purpose of the others

Reply:
To control the default pycodestyle behaviour according to our requirements.

Reply:
when someone established that those are our requirements ???,  since the pr is open and there is no discussion about it

Reply:
The discussion is open, if you want to take part of that, you are welcome. @andrey-canon 
This requested change is just to add a configuration and not let pycodestyle use the default values.

Reply:
there was already a pycodestyle section, however I have added statistics that is the only value that makes any sense at this moment for me

Reply:
Ok, is this reading the default setup.cfg file? Or do you need to add the file path to the command?

Reply:
yes